receiver/splunkhecreceiver/receiver.go

```diff
@@ -72,6 +77,13 @@ var (
 	errInternalServerError    = initJSONResponse(responseErrInternalServerError)
 	errUnsupportedMetricEvent = initJSONResponse(responseErrUnsupportedMetricEvent)
 	errUnsupportedLogEvent    = initJSONResponse(responseErrUnsupportedLogEvent)
+
+	//splunk metadata
+	metadataMap = map[string]string{
+		index:      splunk.DefaultIndexLabel,
```

Comment:
I think this should be `config.HecToOtelAttrs.Index` as this is configurable param of the receiver.

Reply:
ditto on the other attributes